nucypher/policy/conditions/lingo.py

```diff
@@ -122,19 +122,11 @@ def _sanitize_value(self, value):
 
     def _process_data(self, data: Any) -> Any:
         """
-        If a key is specified, return the value at that key in the data if data is a dict or list-like.
-        Otherwise, return the data.
+        Solidity will only return a list
```

Comment:
Careful, Solidity can return a tuple. The prior docstring was good, just needed tweaking.